app/controllers/v0/sessions_controller.rb

```diff
@@ -140,6 +141,25 @@ def build_logout_errors(logout_response, logout_request)
       errors
     end
 
+    # temporarily logging for discovery
+    def additional_logging_for_logout_request(logout_response, logout_request)
+      request_id = begin
+                     JSON.parse(params[:RelayState] || '{}')[:originating_request_id]
```

Comment:
I think this may need to be `['originating_request_id']` rather than `[:originating_request_id]`. I had a similar issue around that when getting the `type` before.

Reply:
GOOD CATCH!